### ISSUE TITLE
Linux Support

### ISSUE THREAD

**Issue by AWildErin** [AWildErin/sdklauncher#5]
Idk what's really required for linux so I'll leave this for somebody else

**Comment by Trico-Everfire** [comment]
I can compile it for linux, is there anything specific that it needs to do?

**Comment by AWildErin** [comment]
Not really, once I push my steam branch can you just verify that actually works and stuff? The current way I was doing exe launching won't work on linux, but that should probably be a fairly easy fix

**Comment by Trico-Everfire** [comment]
> 

can dew

**Comment by Trico-Everfire** [comment]
I realise, isn't this issue resolved? Provided I made steam api compile on Linux?

**Comment by AWildErin** [comment]
Yes but there's other stuff that will eventually need to be tested, like app launching for example
